common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -129,11 +129,23 @@ object WhiskConfig extends Logging {
                 else null
             } else null
 
+        def propfileOpenwhiskHome(): File = {
+            val dir = sys.env.get("OPENWHISK_HOME")
```

Comment:
For your reference: You can rewrite the if-else below as: `dir.map { propfile(_, true) } getOrElse null`.